packages/engine-test-utils/src/__tests__/pods-core/MarkdownPod.spec.ts

```diff
@@ -503,11 +505,21 @@ describe("markdown import pod", () => {
             encoding: "utf8",
           }
         );
+        const fileBody3Content = fs.readFileSync(
+          path.join(vpath, "folder.something.md"),
+          {
+            encoding: "utf8",
+          }
+        );
         expect(fileBody.match("test.txt")).toBeTruthy();
         const assetPath = path.join("assets", "test.txt").replace(/[\\]/g, "/");
+        const imagePath = path
+          .join("assets", "image.JPEG")
+          .replace(/[\\]/g, "/");
```

Comment:
nitpick: `path.posix.join` will always use unix style joins instead if that's the intention. Or just type `assets/image.JPEG` directly :)